assets/src/containers/AssignmentPlanning.js

```diff
@@ -20,83 +22,103 @@ const styles = theme => ({
   paper: {
     padding: theme.spacing.unit * 2,
     color: theme.palette.text.secondary
+  },
+  graded: {
+    width: '10px',
+    height: '10px',
+    background: 'lightskyblue',
+    display: 'inline-block'
+  },
+  ungraded: {
+    width: '10px',
+    height: '10px',
+    background: 'gray',
+    display: 'inline-block'
   }
 })
 
+const getCurrentWeek = assignmentData => {
```

Comment:
I thought the `currentWeek` data should be fetched from `/info`?

Reply:
Assignments view is different the current week info is baked in the assignment data itself when calculating the which assignment belongs to which week.
 I don't know why files view made a decision to get it from a separate backend call. good question!